babel-plugin-dotenv/index.js

```diff
@@ -19,11 +25,8 @@ module.exports = function (data) {
 
                 if (path.node.source.value === options.replacedModuleName) {
                   var config = dotEnv.config({ path: sysPath.join(configDir, configFile), silent: true }) || {};
-                  var platformPath = (process.env.BABEL_ENV === 'development' || process.env.BABEL_ENV === undefined)
-                                          ? configFile + '.development'
-                                          : configFile + '.production';
+                  var platformPath = configFile + (environments[process.env.BABEL_ENV] || environments['development']);
```

Comment:
Cant we use a different ´env´ then `BABEL_ENV` to solve the issue with react-native overridin it?
```javascript
var platformPath = configFile + (environments[process.env.ENV_FILE] ||
```

then you can run or build you app with `$ ENV_FILE =.env.staging npm run ios`

Reply:
My bad didn't see that it was in the babel plugin. So not possible I guess